notebooks/test_output/SampleSheets/Tellseq_samplesheet_instrument_iseq_set_col19to24.csv

```diff
@@ -2,7 +2,7 @@
 Experiment Name,RKLtest_col19to24,,,,,,,,,,
 Investigator Name,Enter the investigator name (optional),,,,,,,,,,
 Project Name,TestProjA_10002_1_2_3_4_10to1dilution,,,,,,,,,,
-Date,2025-04-01,,,,,,,,,,
+Date,2025-09-04,,,,,,,,,,
```

Comment:
This change, while little, is an indication of a drawback of the current testing method.  Some of the files that are produced by the notebook (the kind called "samplesheet" files) include the date  on which they were produced--it will be different every day that they are run!  This means that our method of just checking line-by-line for perfect matches *will always fail* with this kind of file.  It wasn't an issue for notebook D, because notebook D doesn't produce that kind of output file ... but notebook C produces three of them!  This suggests that the testing approach used by notebook D is not robust enough to work for all the other notebooks.